sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ParseDriver.scala

```diff
@@ -151,9 +152,30 @@ object CatalystSqlParser extends AbstractSqlParser {
  * have the ANTLRNoCaseStringStream implementation.
  */
 
-private[parser] class ANTLRNoCaseStringStream(input: String) extends ANTLRInputStream(input) {
+private[parser] class ANTLRNoCaseStringStream(wrapped: CodePointCharStream) extends CharStream {
```

Comment:
Maybe we should rename this to matter match what it is actually doing. `NoCaseCharStream`?